src/Components/Aspire.MySqlConnector/MySqlConnectorCommon.cs

```diff
@@ -0,0 +1,14 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using OpenTelemetry.Metrics;
+
+internal static class MySqlConnectorCommon
+{
+    public static void AddMySqlMetrics(MeterProviderBuilder meterProviderBuilder)
```

Comment:
I'm not sure we need this given we only have 1 component. Is there an EntityFramework "MySqlConnector" library?

Reply:
The EF Core library built on top of MySqlConnector is https://github.com/PomeloFoundation/Pomelo.EntityFrameworkCore.MySql.

Reply:
I'm happy to add support for that too, but will probably do it as a separate PR.